src/test/init.js

```diff
@@ -26,7 +62,7 @@ test('init()', nest => {
     const mutators = ['a', 'b', 'c'].map(
       (i) => (o, { instance }) => { instance[i] = i; });
 
-    const actual = init(...mutators)();
+    const actual = init(mutators[0], mutators.slice(1, 3))();
```

Comment:
To me personally it's super confusing to have a same name of a function with different behavior. I already said it with stampit, but having X different implementations of `compose` seems like a way to unmaintainable hell in my opinion. Call me lazy, but I think this is a bad decision.

Just think of a new user coming to stamps which are already mind blowing on its own. That user has to face decision what module to pick: stamp-specification, stampit or stamp-utils? Studying what are differences and what he might actually need. I believe that for such innovative approach, the entry should be as smooth as possible.

I think that eventually there should be just single most performant implementation of a `compose` and ideally any helper functions could be made as a separate module (I really like babel plugins way).